### PR TITLE
added code from basic portfolio [2]

### PR DESCRIPTION
**Pull request by buelowek1225**

### CODE REVIEW

File: contact.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        <li><a class ="active" href="index.html">Home</a></li>
+        <li><a href="contact.html">Contact</a></li>
+        <li><a href="portfolio.html">Portfolio</a></li>
+      </ul>
+      <div class="title-info">
+        <h1>Contact Me</h1>
+      </div>
+      <div class=form>
+        <form action="/action_page.php"> <!--what does the /action_page.php do-->
+        Name:<br>
```
Comment (wrobe0709) [inline contact.html:24]: Indentation is off

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,208 @@
+/* Downloaded Fonts */
+/* font-family: 'Anton', sans-serif;
+font-family: 'Hind Siliguri', sans-serif;
```
Comment (wrobe0709) [inline css/style.css:3]: Remove commented out code.

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+}
+
+a {
+  color: whitesmoke;
+  font-family: 'K2D', sans-serif;
+  text-decoration: none;
+  margin: 5px;
+}
+
+/* INDEX PAGE */
```
Comment (wrobe0709) [inline css/style.css:51]: You should have different files for different page's css files.

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+
+.title-info {
+  margin: 60px;
+}
+
+h1 {
+  color: whitesmoke;
```
Comment (wrobe0709) [inline css/style.css:58]: Use hex colors.

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+  display: inline;
+  margin: 1%;
+}
+
+.personal {
+  display: flex;
+  margin-left: 20%;
+  margin-right: 20%;
+  margin-top: 5%;
+  background-color: #E1E1E166;
```
Comment (wrobe0709) [inline css/style.css:105]: Use lowercase hex colors. Why are you using a hex color with 8 digits?

**Review by wrobe0709 — CHANGES_REQUESTED** [review]
Please address review comments.

File: contact.html (hunk 1 above)
Comment (buelowek1225) [reply contact.html:24]: I corrected the indentation on line 24 to like 35.

File: css/style.css (hunk 2 above)
Comment (buelowek1225) [reply css/style.css:3]: commented out code removed from style.css file

File: css/style.css (hunk 3 above)
Comment (buelowek1225) [reply css/style.css:51]: I will do this moving forward with my projects for this example I am keeping them the same.

File: css/style.css (hunk 5 above)
Comment (buelowek1225) [reply css/style.css:105]: The last two digits represent opacity.

**Pull request closed without merging**